src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -648,7 +648,12 @@ const FeatureFlagsPage: React.FC = () => {
     >
       <Container padding={{ top: 'medium', right: 'xlarge', left: 'xlarge' }}>
         {FILTER_FEATURE_FLAGS && hasFeatureFlags && (
-          <FlagTableFilters features={features} currentFilter={flagFilter} updateTableFilter={setFlagFilter} />
+          <FlagTableFilters
+            features={features}
+            currentFilter={flagFilter}
+            updateTableFilter={setFlagFilter}
```

Comment:
Actually could you not have added your logic to do the check/set the state here at the top level? instead of you having to prop drill the `setPageNumber` 

```suggestion
            updateTableFilter={(filter)=> { 
                if (Object.keys(filter).length) {
                   setPageNumber(0)
                }
                setFlagFilter(filter)}
            }
```

Reply:
that's a good idea! why the need for `Object.keys` though on top of checking the length?

Reply:
I wasn't sure if `filter` was an object or not - from a glance it looked like it was an object ? if so, if you check an empty object for falsy - e.g `if(filter)` it'll return true:

https://jsfiddle.net/sr3nch2a/

Reply:
Could also just check for `if (!filter.key)`

Reply:
@Kira-Jennings I only see a key property within `queryProps`, I'll tag you in the updated change
![image](https://user-images.githubusercontent.com/95680496/176875088-1c1c0eb4-1eae-4b89-a32d-e7f2cb3cc8da.png)